tests/configuration.py

```diff
@@ -62,6 +62,7 @@
 elasticsearch.port = os.getenv("ES_PORT", "9200")
 elasticsearch.protocol = os.getenv("ES_PROTOCOL", "http")
 elasticsearch.url = os.getenv("ES_URL", f"{elasticsearch.protocol}://{elasticsearch.hosts}:{elasticsearch.port}")
+elasticsearch.kibana_url = os.getenv("KIBANA_URL", "NA")
```

Comment:
Any reason to have "NA" as a default value?

Reply:
I used the same approach [as used](https://github.com/elastic/cloudbeat/blob/3df384ba3b3ec661275d4b434ece3db1a6946315/deploy/test-environments/fleet_api/src/configuration_fleet.py#L34) in fleet_api, how would you suggest to changing it?

Reply:
If you're not addressing the "NA" case in your code directly, I would default to an empty string (which in python evaluates to `false`, so it's a bit safer), I'm not sure about the usage in the other "NA" cases